src/Calendar.Plugin.Sample/SampleApp/ViewModels/AdvancedPageViewModel.cs

```diff
@@ -46,36 +47,34 @@ public AdvancedPageViewModel() : base()
             // with indexer
             Events[DateTime.Now] = new List<AdvancedEventModel>(GenerateEvents(2, "Boring"));
 
-            MonthYear = MonthYear.AddMonths(1);
-
-            Task.Delay(5000).ContinueWith( _ =>
-            {
-                // indexer - update later
-                Events[DateTime.Now] = new ObservableCollection<AdvancedEventModel>(GenerateEvents(10, "Cool"));
-
-                // add later
-                Events.Add(DateTime.Now.AddDays(3), new List<AdvancedEventModel>(GenerateEvents(5, "Cool")));
+            ShownDate = ShownDate.AddMonths(1);
 
-                // indexer later
-                Events[DateTime.Now.AddDays(10)] = new List<AdvancedEventModel>(GenerateEvents(10, "Boring"));
+            Task.Delay(5000).ContinueWith(_ =>
+           {
+               // indexer - update later
+               Events[DateTime.Now] = new ObservableCollection<AdvancedEventModel>(GenerateEvents(10, "Cool"));
 
-                // add later
-                Events.Add(DateTime.Now.AddDays(15), new List<AdvancedEventModel>(GenerateEvents(10, "Cool")));
+               // add later
+               Events.Add(DateTime.Now.AddDays(3), new List<AdvancedEventModel>(GenerateEvents(5, "Cool")));
 
+               // indexer later
+               Events[DateTime.Now.AddDays(10)] = new List<AdvancedEventModel>(GenerateEvents(10, "Boring"));
 
-                Task.Delay(3000).ContinueWith(t =>
-                {
-                    MonthYear = MonthYear.AddMonths(-2);
+               // add later
+               Events.Add(DateTime.Now.AddDays(15), new List<AdvancedEventModel>(GenerateEvents(10, "Cool")));
 
-                    // get observable collection later
-                    var todayEvents = Events[DateTime.Now] as ObservableCollection<AdvancedEventModel>;
+               Task.Delay(3000).ContinueWith(t =>
+               {
+                   ShownDate = ShownDate.AddMonths(-2);
 
-                    // insert/add items to observable collection
-                    todayEvents.Insert(0, new AdvancedEventModel { Name = "Cool event insert", Description = "This is Cool event's description!", Starting = new DateTime() });
-                    todayEvents.Add(new AdvancedEventModel { Name = "Cool event add", Description = "This is Cool event's description!", Starting = new DateTime() });
+                   // get observable collection later
+                   var todayEvents = Events[DateTime.Now] as ObservableCollection<AdvancedEventModel>;
 
-                }, TaskScheduler.FromCurrentSynchronizationContext());
-            }, TaskScheduler.FromCurrentSynchronizationContext());
+                   // insert/add items to observable collection
+                   todayEvents.Insert(0, new AdvancedEventModel { Name = "Cool event insert", Description = "This is Cool event's description!", Starting = new DateTime() });
```

Comment:
Might be good to name these consistently with the other events. We currently have Cool eventX, Cool event insert and Cool event add, its starting to look a bit messy